src/main/resources/db/migration/all/20240117111637__add_assessment_id_to_domain_event.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE domain_events ADD COLUMN assessment_id TEXT;
```

Comment:
Should this be a UUID?

Reply:
Good spot! it absolutely should be. I've updated it to UUID and also explicitly stated its nullable (which is default anyway). Code was already using UUID thankfully :D